testsuite/features/support/constants.rb

```diff
@@ -967,6 +982,10 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sl-micro-6.0' => # TODO: Add  sl-manager-tools-for-micro6-pool-x86_64-6.0   sl-manager-tools-for-micro6-updates-x86_64-6.0 when they are ready
```

Comment:
They seem to be ready. At least I can see them in my HEAD deployment and in SCC, although marked as ALPHA.
![Uploading image.png…]()

Reply:
added